src/components/LocalIncidence.jsx

```diff
@@ -75,60 +20,25 @@ function LocalIncidence ({ values, date, setDate, className, isMobile = false, l
       }}
     >
       {({ measureRef }) => (
-        <div ref={measureRef} className={classNames('grid grid-rows-3 gap-2', className)}>
-          <Chart
-            colors = {colors}
-            width={width}
-            isMobile={isMobile}
-            heading='Incidence'
-            controls={
-              <Checkbox
-                id='lambda_display_type'
-                checked={incidence === 'area'}
-                label='Stack'
-                onChange={handleLambdaChange}
-                toggle
-              />
-            }
-            area_data={area_data}
-            date={date}
-            setDate={setDate}
-            parameter='lambda'
-            type={incidence}
-            stroke={lineColor}
-          />
-          <Chart
-            colors= {colors}
-            width={width}
-            isMobile={isMobile}
-            heading='Proportion'
-            controls={
-              <Checkbox
-                id='proportion_display_type'
-                checked={proportion === 'area'}
-                label='Stack'
-                onChange={handlePropChange}
-                toggle
-              />
-            }
-            area_data={area_data}
-            date={date}
-            setDate={setDate}
-            parameter='p'
-            type={proportion}
-            stroke={lineColor}
-          />
-          <Chart
-          colors = {colors}
-            width={width}
-            isMobile={isMobile}
-            heading='R'
-            area_data={area_data}
-            date={date}
-            setDate={setDate}
-            parameter='R'
-            stroke={lineColor}
-          />
+        <div ref={measureRef} className={classNames(`grid grid-rows-${numCharts} gap-2`, className)}>
```

Comment:
There's a small tech issue here, dynamic classes don't go well with Tailwind ([link](https://tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html)). If you inspect the element in the deployment, the `grid-rows-*` class is not in the bundle. Luckily it looks like the default behaviour of a grid is to split into even rows, so I think `grid-rows-*` can be removed - unless you know a case where it's not working?

Reply:
p.s. these classes might help as they don't depend on specific numbers: https://tailwindcss.com/docs/grid-auto-rows

Reply:
Oh yeah - sorry that's obvious in hindsight :) Thanks for the catch!